hikey_debian.mk

```diff
@@ -226,7 +226,7 @@ PERF_FLAGS := CROSS_COMPILE_HOST=$(CROSS_COMPILE_NS_USER) \
 	TA_DEV_KIT_DIR=$(OPTEE_OS_TA_DEV_KIT_DIR)
 
 aes-perf: optee-os optee-client
-	$(MAKE) -C -j$(NPROC) $(AESPERF_PATH) $(PERF_FLAGS)
+	$(MAKE) -j$(NPROC) -C $(AESPERF_PATH) $(PERF_FLAGS)
```

Comment:
I can't argue against that, but it's a question about maintaining it also. That was one of the reasons why we moved it to xtest and the second was to have it all in the same test application.